### ISSUE TITLE
[FEATURE] Add "used" tracking for reactions

### ISSUE THREAD

**Issue by Eranziel** [Eranziel/foundryvtt-lancer#561]
**Is your feature request related to a problem? Please describe.**
It can be easy to lose track of which reactions have been used in a given round. Especially for the GM, who has to track multiple NPCs.

**Describe the solution you'd like**
Add uses tracking for all reactions (default to 1 use/round), with UI element to toggle, and automatically refresh all reactions at the start of an actor's turn. It would behave very similar to the "charged" tracking for NPC features, but without the roll to refresh.

Bonus points for being able to edit the number of uses per round for the few cases that give multiple reactions.

The same could be applied to systems with the "X/round" tag.

**Describe alternatives you've considered**
Just do it in your head.

**Comment by msprijatelj** [comment]
Honestly, since Reactions usually have the 1/round tag, it may be more straightforward to immediately start checking for that 1/round tag? Though having a default behavior expecting 1/round specifically on reactions would definitely be reasonable.

**Comment by Eranziel** [comment]
Agreed, if the X/round tag is present we should use that value instead. It would also be fairly straightforward to also add this to X/round items which are not reactions.

I think the biggest UX problem to solve for this is to distinguish it from limited uses. Maybe label it "available" or something, instead of "uses"?

**Comment by Eliemer** [comment]
Reactions only refresh at the start of a unit's first turn that round. So, templates like Ultra and Elite don't refresh reactions every activation

**Comment by msprijatelj** [comment]
Well, Ultras do at least, but correct on Elites!
